docs/normalize.md

```diff
@@ -0,0 +1,66 @@
+# The `normalize()` function
+
+Each base class defined using the NOMAD schema has a set of public functions which can be used at any moment when reading and parsing files in NOMAD (see their corresponding documentation pages). The only special situation happens for the `normalize(archive, logger)` function.
+
+This function is ran within the NOMAD infrastructure by the [`MetainfoNormalizer`](https://github.com/nomad-coe/nomad/blob/develop/nomad/normalizing/metainfo.py) in the following order:
+
+1. The deepest child class `normalize()` function is ran before their parents' `normalize()` function.
+2. For sibling sections, the `normalize()` function is executed from the smaller to the larger `normalizer_level` attribute. If `normalizer_level` is not set up, the order is established by the attributes definition order in the parent class.
```

Comment:
"not set up" --> "not set"?

Also, what if they are set to be the same, I guess then it defaults again to the order in the parent class?

Reply:
> Also, what if they are set to be the same, I guess then it defaults again to the order in the parent class?

Yes, I added "If `normalizer_level` is not set or if they are the same for two different sections, the order..."